src/Formik.tsx

```diff
@@ -921,3 +942,46 @@ function arrayMerge(target: any[], source: any[], options: any): any[] {
   });
   return destination;
 }
+
+/** Return multi select values based on an array of options */
+function getSelectedValues(options: any[]) {
+  const result = [];
+  if (options) {
+    for (let index = 0; index < options.length; index++) {
+      const option = options[index];
+      if (option.selected) {
+        result.push(option.value);
+      }
+    }
+  }
+  return result;
+}
+
+/** Return the next value for a checkbox */
+function getValueForCheckbox(
+  currentValue: string | any[],
+  checked: boolean,
+  valueProp: any
+) {
+  if (valueProp == 'true' || valueProp == 'false') {
+    return !!checked;
+  } else {
+    if (checked) {
+      return Array.isArray(currentValue)
+        ? currentValue.concat(valueProp)
+        : [valueProp];
+    } else {
+      if (!Array.isArray(currentValue)) {
+        return !!currentValue;
+      }
+      const index = currentValue.indexOf(valueProp);
```

Comment:
you can use `includes` to instead of `indexOf`

Reply:
I need to reuse the index.